### PR TITLE
New resource-topics landing page

### PR DESCRIPTION
**Pull request by nick-mon1**
## Summary
Redesign for the the [resources landing page](https://digital.gov/resources/) to be organized by topics.

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/nl-update-topics-page/resources/)

<details><summary>Screenshot</summary>
<p>

Guides and Resources _ Digital_gov · 11 11am · 05-17

</p>
</details> 

### How To Test

1. Visit /resources and see new page layout
2. Should see the following resource categories (left to right)

| category |
|--------|
| Design |
| Data & analysis |
| Operations |
| Content & communication |
| Technology | 
| Strategic development | 

3. Topics are in sentence case and listed in alphabetical order
4. Topic links work as expected

---

### Dev Checklist
- [x] Submit https://digital.gov/resources/ to wayback machine
- [x] PR has correct labels
- [x] A11y testing (voice over testing, meets WCAG, run axe tools)
- [x] Code is formatted properly

### CODE REVIEW

**Review by bonnieAcameron — COMMENTED** [review]
We are missing a few topics (re: discussion on the original card): 

Contact Centers in Operations
Open Government in Data & Analysis
Policies and Regulations in Strategic Development

File: themes/digital.gov/layouts/resources/list.html [hunk 1]
```diff
@@ -1,154 +1,38 @@
...
+        <p>
+          When you have a requirement to meet or an improvement to introduce to
+          your team or product, but you don’t know where to start: start here.
+          We have resources at every level available by collection so you can
+          explore how to meet the requirements of web legislation.
+        </p>
+      </header>
+
+      <div class="usa-section">
+        <h2>Browse by topic</h2>
```
Comment (bonnieAcameron) [inline themes/digital.gov/layouts/resources/list.html:26]:
```suggestion
        <h2>Browse by topics</h2>
```

**Review by bonnieAcameron — CHANGES_REQUESTED** [review]
Image 5-20-24 at 4 59 PM

1. Can we close the space at the top of the "browse by topic section"? In my image I used a .5rem padding-top
2. Can we add a bit of contrast in the page header? Ideally the "light gray"ish color (I used #edeff0) would extend all the way across the page, but I don't have enough skill to mock that up.
3. Would we be following our style if we introduced a bit more color contrast between the topic items and the topic list headings?

**Comment by sarah-sch** [comment]
@bonnieAcameron and @nick-mon1 - I'm wondering if we can review the intro text to introduce shorter, simpler sentences for plain language. @bonnieAcameron, what do you think of this suggestion:

Don't know where to start with a federal requirement? Aren't sure how to adopt a new technology with your team? Start here! We have resources for every level. Our collections are organized by popular topics, and they will help you meet the requirements of federal web legislation.

**Review by sarah-sch — CHANGES_REQUESTED** [review]
Suggesting new language for the intro:
Don't know where to start with a federal requirement? Aren't sure how to adopt a new technology with your team? Start here! We have resources for every level. Our collections are organized by popular topics, and they will help you meet the requirements of federal web legislation.

**Comment by nick-mon1** [comment]
@bonnieAcameron 

1. The space between the **Browse by topic** and header section has been reduced
2. We should discuss adding the option of adding a background to one of our template clean up cards. Also, we do not use that pattern anywhere else and we should keep landing pages consistent.
3. Similar to the communities page, I've picked some colors for the headers to add some color. Feel free to pick something different.
4. Feel free to create a commit for @sarah-sch suggestions

Screenshot 2024-05-22 at 4 01 04 PM

**Review by bonnieAcameron — COMMENTED** [review]
The tab name still says "Guides and Resources." Since we're about to have an independent guides landing page, can we update this to just "Resources"?

Image 6-3-24 at 3 51 PM

**Comment by bonnieAcameron** [comment]
> @bonnieAcameron
> 
> 1. The space between the **Browse by topic** and header section has been reduced
> 2. We should discuss adding the option of adding a background to one of our template clean up cards. Also, we do not use that pattern anywhere else and we should keep landing pages consistent.
> 3. Similar to the communities page, I've picked some colors for the headers to add some color. Feel free to pick something different.
> 4. Feel free to create a commit for @sarah-sch suggestions
> 
> Screenshot 2024-05-22 at 4 01 04 PM

This definitely needed color. I'm wondering if we can use blue-warm-50v (#2672de) for all of the heading colors to adhere more to the style guide?

**Review by bonnieAcameron — APPROVED** [review]

**Review by mejiaj — CHANGES_REQUESTED** [review]
The PR description says resources redesign, can we update to Topics?

Overall the new page looks wonderful, nice job! Comments below for maintainability & clarity.

File: themes/digital.gov/src/scss/new/_resources.scss [hunk 2]
```diff
@@ -1,25 +1,48 @@
...
+  &__list-item {
+    padding-bottom: units(1);
+  }
+
+  &__link {
+    color: color("blue-70v");
+    // allows first-letter psuedo element to target anchor tags
+    display: inline-block;
+    text-decoration: none;
+    text-transform: lowercase;
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply themes/digital.gov/src/scss/new/_resources.scss:38]: Plain language rules suggest to use Capital casing for for topic names. Emphasized by Bonnie's requirements in the figma doc.

File: themes/digital.gov/src/scss/new/_resources.scss [hunk 3]
```diff
@@ -1,25 +1,48 @@
...
+
+    &::first-letter {
+      text-transform: uppercase;
+    }
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply themes/digital.gov/src/scss/new/_resources.scss:47]: Kept to be consistent with sentence casing.

File: themes/digital.gov/layouts/partials/core/topics-list.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{{/* Display a header and list of topic links
+
+  @param {string} .data.topics - A list of topic strings to render a link
+  @param {string} .data.color_label - A label for the resource-topic header to display a color, see _resources.scss for values
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply themes/digital.gov/layouts/partials/core/topics-list.html:4]: Not anymore, removed.

File: themes/digital.gov/src/scss/new/_resources.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (nick-mon1) [reply themes/digital.gov/src/scss/new/_resources.scss:38]: See comments below for some additional clarification.

**Review by RileySeaburg — DISMISSED** [review]
I don't have any hard opinions on this. 
Either way you work out with @mejiaj to order the data or present the styles works for me. 
Let's make sure we maintain consistency throughout the project.

**Comment by nick-mon1** [comment]
Some questions on the sentence casing for the topics titles and links for @bonnieAcameron and @mejiaj 

- Topic file (topic/index.md) title field sets the display text
- Individual topics pages display the topic name with no casing styling as the page title
- The resources page displays each topic title with no casing styling as a link


1. The topic markdown file should be the canonical source for displaying the title
2. if so, are there ever cases where we want to display it in a different casing for titles, links, etc...?
3. For some topics that use an acronym (i.e. API), we don't use that in the title but is displayed in the figma requirements, let's follow this up with additional work if needed.

File: data/topics/content_and_communications.yml [hunk 5]
```diff
@@ -0,0 +1,10 @@
+category: Content & communication
+topics:
+  - name: communication
+  - name: content strategy
+  - name: multilingual
+  - name: multimedia
+  - name: plain language
+  - name: podcast
+  - name: social media
+  - name: trust
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply data/topics/content_and_communications.yml]: Moved these the front matter section on `resources/_index.md`.

File: data/topics/content_and_communications.yml [hunk 6]
```diff
@@ -0,0 +1,10 @@
+category: Content & communication
+topics:
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply data/topics/content_and_communications.yml]: Will move this work to a new card to use hugo page bundles.

**Comment by bonnieAcameron** [comment]
We are also missing a topic: User Experience

File: themes/digital.gov/src/scss/new/_resources.scss [hunk 7]
```diff
@@ -1,25 +1,31 @@
...
-  font-weight: 400;
-  @include u-border-right("1px", "gray-90", "solid");
+  &__link {
+    // allows first-letter psuedo element to target anchor tags
+    display: inline-block;
+
+    &:hover {
+      text-decoration: underline;
+    }
+  }
```
Comment (mejiaj) [inline themes/digital.gov/src/scss/new/_resources.scss:30]: Are these still needed? Not seeing a difference when they're disabled.

**Review by mejiaj — COMMENTED** [review]
Minor question, otherwise LGTM.

File: themes/digital.gov/src/scss/new/_resources.scss (hunk 7 above)
Comment (nick-mon1) [reply themes/digital.gov/src/scss/new/_resources.scss]: Removed

**Review by mejiaj — APPROVED** [review]
Thanks for the updates.

**Tested**
- [x] Small and large screen sizes. 
- [x] Axe page scan - No errors found.
- [x] Styles are minimal and redundant rules removed.

**Comment by nick-mon1** [comment]
@ToniBonittoGSA I'm able to render an `m-dash` on my local build with VSCode.

Screenshot 2024-07-02 at 9 56 37 AM

**Review by ToniBonittoGSA — APPROVED** [review]

**Pull request merged**